universal-application-tool-0.0.1/test/repository/ProgramRepositoryTest.java

```diff
@@ -2,24 +2,29 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableList;
 import models.Program;
 import org.junit.Test;
+import services.program.ProgramDefinition;
 
 public class ProgramRepositoryTest extends WithPostgresContainer {
 
   @Test
   public void createProgram() {
     final ProgramRepository repo = app.injector().instanceOf(ProgramRepository.class);
-    Program program = new Program();
-    program.setName("program one");
-    program.setVersion(1L);
-    program.setObject(ImmutableMap.of("key", "value"));
+
+    ProgramDefinition definition =
+        ProgramDefinition.builder()
+            .setId(10L)
```

Comment:
issue: I think we want to avoid setting IDs and instead allow ebean to generate them for us. Doesn't seem like a big deal here, but this can easily lead to ID conflicts since the same database instance is used across all tests. (I know this PR is already merged, but something to consider in the future).